lib/x11utils.pm

```diff
@@ -311,14 +313,32 @@ sub select_user_gnome {
     }
 }
 
-=head2 turn_off_kde_screensaver
+=head2 turn_off_plasma_screen_energysaver
 
- turn_off_kde_screensaver();
+ turn_off_plasma_screen_energysaver()
 
-Turns off the screensaver on KDE
+Turns off the Plasma desktop screen energy saving.
 
 =cut
-sub turn_off_kde_screensaver {
+sub turn_off_plasma_screen_energysaver {
+    x11_start_program('kcmshell5 powerdevilprofilesconfig', target_match => [qw(kde-energysaver-enabled energysaver-disabled)]);
+    if (match_has_tag('kde-energysaver-enabled')) {
+        assert_and_click('kde-disable-energysaver');
+    }
+    assert_screen 'energysaver-disabled';
+    # Was 'alt-o' before, but does not work in Plasma 5.17 due to kde#411758
+    send_key('ctrl-ret');
```

Comment:
```suggestion
    send_key 'ctrl-ret';
```

Reply:
Could (some) of these be added to perltidy? And if they are they apparently were not caught.

Reply:
would be nice but I would not know how